SafeguardSessions/README.md

```diff
@@ -13,40 +13,41 @@
 
 ## About the project
 
-The aim of One Identity Safeguard Power BI Connector (Connector) is to provide a solution for customers to visualize their audit data captured by [One Identity Safeguard for Priviledged Sessions] (SPS) in a highly configurable way compared to the on-box reporting system of SPS. The Connector uses basic authentication with local login method to connect to SPS and imports sessions metadata that matches the criteria given by the Connector's input parameters. After successful data retrieval, the Connector returns with two tables:
+The aim of One Identity Safeguard Power BI Connector (Power BI Connector) is to provide a solution for customers to visualize their audit data captured by [One Identity Safeguard for Priviledged Sessions] (SPS) in a highly configurable way compared to the on-box reporting system of SPS. The Power BI Connector uses basic authentication with the local login method to connect to SPS, and imports sessions metadata that matches the criteria specified in the Power BI Connector's input parameters. After successful data retrieval, the Power BI Connector returns two tables:
 
-- **Info**: contains information about the data fetch process for debugging purposes
+- **Info**: contains information about the data fetching process for debugging purposes
 - **Sessions**: contains the actual sessions metadata
 
-The project also a includes a Power BI report template to quickstart report creation and visualize your audit data.
+The project also a includes a Power BI Report Template (Report Template) to quickstart report creation and visualize your audit data.
```

Comment:
Thank you for the recognition. `One Identity Safeguard` was indeed missing, updated it.

The reason for the `(Report Template)` is is because we consider the full name of our template to be One Identity Safeguard Power BI Report Template and we refer to it onwards as Report Template.